src/app/components/TransactionsTable/index.tsx

```diff
@@ -76,26 +80,65 @@ export default function TransactionsTable({ transactions }: Props) {
                           ~{tx.totalAmountFiat}
                         </p>
                       </div>
-                      <Disclosure.Button className="block h-0 mt-2 text-gray-500 hover:text-black dark:hover:text-white transition-color duration-200">
-                        <CaretDownIcon
-                          className={`${open ? "rotate-180" : ""} w-5 h-5`}
-                        />
-                      </Disclosure.Button>
+                      {([tx.type && "sent", "sending"].includes(tx.type) ||
```

Comment:
when having an a class object... I would write here something like `tx.hasMetadata?` (in ruby) 

but we probably need this check only here, so it's fine.